### PR TITLE
Added NoPCB Pixel

### PR DESCRIPTION
**Pull request by patatman**
This project got me really excited, and I wanted to create it with parts I already have laying around. 
So I modified the STL a bit so I can fit a Wemos D1 mini laying down, and have some room for the headers. 

If this is not what you anticipated for the project, then please delete the PR. 

I've already printed this model on a Ender V2, and the Wemos fits snuck and doesn't need additional mounts to keep it in place.

I also have an edit on the topscreen/mount so you don't need screws but it just pops on like a lid. The downside of this is that you need a fairly good calibrated printer for it to work. The lid can come loose if you yank on it, but with normal use I don't expect that to happen.

Let me know what your thoughts are 👍

### CODE REVIEW

**Comment by ioios-io** [comment]
Hi Patatman, I appreciate and very much welcome your input. I'm not sure if you are aware but the PCB that this model uses already has a D1 Mini on the underside. However, assuming that you have made this modification so that you can use the case without the PCB, I'm happy to include your design as a 'community remix'. I've stolen that term from Thingiverse I think but it describes the category quite well.
Can you confirm that your modifications enable a user to assemble a finished unit minus the custom PCB?
Thanks for your interest and your time in submitting this, it's appreciated. :)
John Lumley

For clarity, the animation in the Pithy Pixel README.md shows a glimpse of the underside D1 Mini even though it's the screen version rather than the pixel (they are very similar underneath). https://github.com/ioios-io/pithy/tree/main/Cases/stl/Pithy%20Pixel

**Comment by patatman** [comment]
Thanks for your reply! 
And yes, I'm aware that it normally uses a PCB, but I was a little too late for the Kickstarter. 
But since you opensourced everything, I could easily recreate it without the PCB.

I like the 'community remix' tag, I think it also gives a clear indication that it's from the Community and not the original project. 

>Can you confirm that your modifications enable a user to assemble a finished unit minus the custom PCB?

I'm still waiting om some parts, but I think I have sufficient room for the internals so it should be a nice fit.
In short: Yes, this allows for a ioio Pixel without the custom PCB. 

Although this is a PR for the Pixel version, I'm also waiting on some screens to replicate the screen version 😄 
As soon as I have that one build, I'll make a new PR with modifications (if needed at all).

**Comment by ioios-io** [comment]
That's great :) I've just added a readme to the root of the pithy repo and included at the bottom some contributing guidelines. Since you're the first collaborator (and an experienced github user), could you take a quick look at them and ensure I'm being clear and reasonable with my wording? And I was perhaps thinking 'community' would be a better directory name than 'remixes' since I'd like to encourage entirely new designs too. What do you think?
And can't wait to see the screen one too! :D 
John

**Comment by patatman** [comment]
The readme looks perfect! I think it's very clear, and I'll make sure to adjust my PR accordingly 😄 
And I agree, community sounds more friendly. 

Should the PR's become to much, you can always add a PR template like [Home Assistant](https://github.com/home-assistant/home-assistant.io/tree/current/.github)
But for now that's a bit overkill I guess, since we're the only contributors.

#offtopic
Some amazing news for Esphome this evening, they've been acquired by Home Assistant! I think this will ensure it stays open-source and contributed for the foreseeable future.

**Comment by patatman** [comment]
I've just moved the files around, I'll also make sure to post a make picture once it's done.
This way people will now the end-result 🙂

**Comment by ioios-io** [comment]
That HA PR template is very useful, I hadn't seen that before thanks. :) 
And yeah, was just reading about the ESPHome thing. Someone on a Facebook group called it 'Nabu Casa aquires ESPHome' and so cue the nonsense. I think and hope it will work out very well indeed. Can't see them paywalling it and I can see loads of cool advantages that they will bring. Happy days :)

**Comment by ioios-io** [comment]
How about your dir being called NoPCB Pixel or PCBless Pixel?  Something that highlights the 'no board' factor.

**Comment by patatman** [comment]
I can do that tomorrow morning. 
As far as I'm aware you can also make changes to my fork (I left the 'maintainer can make changes' on). If you want it done this evening.
Otherwise I'll fix it in the morning.

**Comment by ioios-io** [comment]
Cool, I thought I'd be able to, still learning PRs! I'll have a play and see what I can do. 
Thanks!

**Comment by patatman** [comment]
Awesome! Looks like you've fixed the PR 😄
